test/regress/regress0/smtlib/set-info-status.smt2

```diff
@@ -3,7 +3,7 @@
 ; EXPECT: sat
 ; EXPECT: unsat
 ; EXPECT-ERROR: Expected result unsat but got sat
-; ERROR-SCRUBBER: sed -e '/Fatal failure within.*/d'
+; ERROR-SCRUBBER: sed -e '/Fatal failure within.*/d' -e '/qemu: uncaught target signal.*/d'
```

Comment:
Could we move that into the general regression script? This is not something specific to this regression, but would always occur when exit code is non-zero, right?

Reply:
Done [2c54794](https://github.com/cvc5/cvc5/pull/8268/commits/2c54794a4ed28b32303d71dc5148ea829bf0602b)